assets/queries/k8s/cpu_limits/query.rego

```diff
@@ -0,0 +1,32 @@
+package Cx
+
+CxPolicy [ result ] {
+    document := input.document[i]
+    specInfo := getSpecInfo(document)
+    metadata := document.metadata
+
+    containers := document.spec.containers
+    
+    
+    object.get(containers[index]["resources"]["limits"], "cpu", "undefined") == "undefined"
```

Comment:
the query should detect when either _limits_ or _resource_ are not present in the file, e.g. if you remove the limits or resources attribute in the *negative.yaml* the query is still passing, when it should not. I suggest testing these different edge cases.